### PR TITLE
Add persona service

### PR DESCRIPTION
**Pull request by sfitz42**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -25,4 +25,5 @@ export CARBON_COPY_EMAIL_ADDRESS="" # Email address for carbon copy for use with
 # You may need to override base urls if you are running inside WSL on Windows
 
 # export DOCUMENTS_API_BASE_URL="host.docker.internal:3001"
-# export RESPONSES_API_BASE_URL="host.docker.internal:3004"
+# export KNOWLEDGE_API_BASE_URL="host.docker.internal:3003"
+# export RESPONSES_API_BASE_URL="host.docker.internal:3004"
```
Comment (BarryGeeDefra) [inline .env.example]: needs a line break

**Review by BarryGeeDefra — COMMENTED** [review]
Just a missing line break bit otherwise LGTM

File: .env.example (hunk 1 above)
Comment (sfitz42) [reply .env.example]: Done

**Review by BarryGeeDefra — APPROVED** [review]
LGTM

**Pull request merged**